### PR TITLE
OPSEXP-1853: move named template back in chart

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-1853

### CODE REVIEW

File: charts/alfresco-search-service/values.yaml [hunk 1]
```diff
@@ -12,7 +12,6 @@ nodeSelector: {}
 # alfrescoRegistryPullSecrets has to be set
 # More information can be found on
 # https://github.com/Alfresco/acs-deployment/blob/master/docs/helm/registry-authentication.md
-type: search-services
```
Comment (gionn) [inline charts/alfresco-search-service/values.yaml:15]: what are we trying to achieve here?

L5 document this field

File: charts/alfresco-search-service/templates/_helpers.tpl [hunk 2]
```diff
@@ -31,7 +42,11 @@ Get Alfresco Search container Port ("internal")
 Get Alfresco Search Service Port
 */}}
 {{- define "alfresco-search.svcPort" -}}
-{{- .Values.service.externallPort | default 80 | int -}}
+{{- $svcPort := 80 }}
+{{- if hasKey .Values "service" }}
+  {{- $svcPort := default 80 .Values.service.externalPort }}
```
Comment (gionn) [inline charts/alfresco-search-service/templates/_helpers.tpl:47]: no need to default, it's already 80

File: charts/alfresco-search-service/templates/_helpers.tpl [hunk 3]
```diff
@@ -16,6 +16,17 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Alfresco Search Service Host
+*/}}
+{{- define "alfresco-search.host" -}}
+{{- if index .Values.enabled -}}
```
Comment (gionn) [inline charts/alfresco-search-service/templates/_helpers.tpl:23]: what is this top level enabled that I don't see in the values file?

**Review by gionn — CHANGES_REQUESTED** [review]

File: charts/alfresco-search-service/templates/_helpers.tpl (hunk 3 above)
Comment (alxgomz) [reply charts/alfresco-search-service/templates/_helpers.tpl:23]: true that's a blind copy/paste from alfresco-common

File: charts/alfresco-search-service/templates/_helpers.tpl (hunk 2 above)
Comment (alxgomz) [reply charts/alfresco-search-service/templates/_helpers.tpl:47]: well I still want to cover for cases where `.Values.service.externalPort` is not set, but I guess you'll prefer this other way of doing things

File: charts/alfresco-search-service/values.yaml (hunk 1 above)
Comment (alxgomz) [reply charts/alfresco-search-service/values.yaml:15]: From what I could see this is not used anywhere

Comment (alxgomz) [reply charts/alfresco-search-service/values.yaml:15]: I thought it was unused and missed it actually still was

**Review by gionn — APPROVED** [review]
But please have a proper PR title

**Pull request merged**